setup.py

```diff
@@ -51,7 +51,7 @@ def main():
         },
         scripts=[],
         zip_safe=False,
-        install_requires=[],
+        install_requires=['requests', 'PIL'],
```

Comment:
We have been very careful not to force any required packages for this project (thusly making it as slim as possible). I would much rather say that these are only required if you want to use this new mixin. Thoughts?